dev-python/lxml/lxml-4.9.2-r1.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
I don't think you need to revbump anything to add pypy3 since the target is ~arch only. Also it makes reviewing these changes a real pain.

Reply:
Sorry, I learned yesterday I needed to revbump, then I didn't think about exceptions for this rule.

Reply:
For review, have you tried something like `git log --stat -U3 --find-copies-harder` ?

Reply:
Roughly, adding PYTHON_TARGETS is done in place unless 1) you're adding patches that change installed files for existing targets, 2) you're adding them to a stable version and dependencies don't have matching target in the stable version (i.e. you get `pkgcheck scan` failure).